package.json

```diff
@@ -53,26 +53,35 @@
     "url": "git@github.com:MonsantoCo/ol-kit.git"
   },
   "dependencies": {
+    "@material-ui/core": "^4.9.11",
+    "@material-ui/icons": "^4.9.1",
     "@turf/bbox": "^6.0.1",
     "@turf/centroid": "^6.0.2",
     "@turf/helpers": "^6.1.4",
     "geostyler-openlayers-parser": "0.21.0",
+    "lodash.clonedeep": "^4.5.0",
     "lodash.debounce": "~4.0.8",
-    "nanoid": "~2.1.6",
+    "lodash.escaperegexp": "^4.1.2",
+    "material-ui": "^0.20.2",
```

Comment:
is this one also needed?

Reply:
hmmm, I honestly don't know what that is but I don't think it should be there